pylearn2/train_extensions/wmape_channel.py

```diff
@@ -0,0 +1,111 @@
+"""
+TrainExtension subclass for calculating Weighted Mean Average Percentage
+Error scores for regression models on monitoring dataset(s), reported via
+monitor channels.
+:math:`WMAPE = \frac{\sum_i|R_i - P_i|}{\sum_i |R_i|}`
+"""
+
+__author__ = "Junbo chen"
+__copyright__ = "Copyright 2015, Alibaba Group"
+__license__ = "3-clause BSD"
+__maintainer__ = "Junbo Chen"
+
+from theano import config
+from theano import tensor as T
+
+from pylearn2.train_extensions import TrainExtension
+
+
+class WMapeNumeratorChannel(TrainExtension):
+    """
+    Adds a WMape Numerator channel to the monitor for each monitoring dataset.
+    It calculates the numerator of the WMAPE formula.
+
```

Comment:
Could you put the formula here? It is awkward to have to look for what "the numerator of the WMAPE formula" is.
Also, maybe add a "See also" section mentioning `WMapeDenominatorChannel`.